modules/index/src/main/java/org/apache/ignite/internal/index/IndexBuildController.java

```diff
@@ -244,7 +258,7 @@ private void tryScheduleBuildIndex(
      * @param replicaId Replica ID.
      */
     private void stopBuildingIndexesIfPrimaryExpired(TablePartitionId replicaId) {
```

Comment:
The same, shouldn't be there new `ZonePartitionId`?

Reply:
As an additional question: isn't one of subtask is to get rid of `TablePartitionId`?